contributing.MD

```diff
@@ -0,0 +1,55 @@
+To set up a test intance of Icarus, here's what you'll need to do:
+
+# Dependencies
+- [MongoDB](https://www.mongodb.com/) ***(version?)***
```

Comment:
MongoDB should be clarified a bit. Figuring out what version would be good. Giving basic instructions on making a database or account or whatever is needed. This is especially helpful for someone who's never used mongo (like me 😅).

Reply:
Would you be willing to write some info on MongoDB setup?

Reply:
I would be, If I understood it at all 😅